beamer/agent.py

```diff
@@ -99,13 +108,15 @@ def start(self) -> None:
         beamer.metrics.init(self._config)
         self._event_processor.start()
         self._contract_monitor_l2a.start()
+        self._contract_monitor_l2a_registry.start()
```

Comment:
Maybe a renaming to `_request_manager_monitor...` , `_fill_manager_monitor` would make sense now for a better readability of the code

Reply:
Replaced this by multi-contract monitors now.